api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -111,7 +111,8 @@ public class MailServiceImpl implements MailService {
   private static final String TWO_STEP_VERIFICATION = "Turn on Google 2-Step Verification";
   private static final String CT_INSTITUTION_CHECK = "Check that %s allows Controlled Tier access";
   private static final String ERA_COMMON = "Connect your eRA Commons account";
-  private static final String RT_TRAINING = "Complete All of Us Registered Tier Training";
+  private static final String RT_TRAINING =
+      "Complete All of Us Responsible Conduct of Research Training";
```

Comment:
See [this message](https://pmi-engteam.slack.com/archives/C02MWP2RN5P/p1696942677700569?thread_ts=1696878015.979769&cid=C02MWP2RN5P).